docker-compose.yml

```diff
@@ -2,21 +2,23 @@ version: "3.9"
 
 services:
     fosscord:
-        container_name: fosscord
-        image: fosscord
         restart: on-failure:5
         build: .
         ports:
             - "3001-3005:3001-3005"
         volumes:
             - ./:/srv/fosscord-server/
         environment:
-            THREADS: ${THREADS:-1}
+            THREADS: 1
```

Comment:
Since docker compose supports .env files, it's not recommended to hardcode the env var value.